.vib/discourse/cypress/cypress/integration/discourse_spec.js

```diff
@@ -0,0 +1,64 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows to log in and out', () => {
+  cy.login();
+  cy.contains('a', 'Unread');
```

Comment:
I guess this is checking for the unread notifs. My concern here is what would happen if the test is executed twice, would the notifs still be unread?

Maybe we can change this check to something that does not depend on a status (`read/unread`) if the goal here is to ensure we are logged in. WDYT?

Reply:
In this case, it is an additional view that only appears when you are logged in and shows you your unread topics as shown below. There are other elements only present when authenticated, but this one shouldn't give us any issue.

<img width="907" alt="Screenshot 2022-05-18 at 13 54 44" src="https://user-images.githubusercontent.com/26460210/169032944-45485e04-c3d6-4987-afc5-90d907d059e5.png">

Reply:
Ok, got it!